change/@uifabricshared-theming-react-native-2020-04-09-11-38-21-fixmactheming.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "🐛 Make ThemingModule not crash on Mac",
```

Comment:
nit: its really *all* platforms will RedBox currently unless the Theming native module is linked.